NOTICE

```diff
@@ -0,0 +1,9 @@
+Apache Maven Wrapper
+Copyright 2020-2022 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+The Initial Developers of maven-wrapper module, which is copied from, derived from,
+or inspired by Gradle Wrapper, are Hans Dockter and Adam Murdoch.
+Copyright 2007 the original author or authors.
```

Comment:
This makes it sound like Hans and Adam were the initial developers of the maven-wrapper module, i.e. as if they set out and wrote it for Maven.

```suggestion
The original idea and initial implementation of the maven-wrapper module is derived 
from the Gradle Wrapper which was written originally by Hans Dockter and Adam Murdoch.
Copyright 2007 the original author or authors.
```